public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React Redux App</title>
+    <title>React App</title>
```

Comment:
- [ ] **(OPTIONAL)** It would be nice if you gave this a nice personalized title based on your app.

Reply:
Done!